Source/Core/Core/ActionReplay.cpp

```diff
@@ -288,10 +281,14 @@ void SaveCodes(IniFile* local_ini, const std::vector<ARCode>& codes)
 {
   std::vector<std::string> lines;
   std::vector<std::string> enabled_lines;
+  std::vector<std::string> disabled_lines;
+
   for (const ActionReplay::ARCode& code : codes)
   {
-    if (code.active)
+    if (code.enabled)
       enabled_lines.emplace_back("$" + code.name);
+    else if (code.default_enabled)
+      disabled_lines.push_back("$" + code.name);
```

Comment:
```suggestion
      disabled_lines.emplace_back("$" + code.name);
```
nitpick, but consistency :P